examples/mnist_with_tensorboard.py

```diff
@@ -0,0 +1,172 @@
+from __future__ import print_function
+from argparse import ArgumentParser
+import logging
+
+from torch.autograd import Variable
+from torch.utils.data import DataLoader
+from torch import nn
+import torch.nn.functional as F
+from torch.optim import SGD
+from torchvision.datasets import MNIST
+from torchvision.transforms import Compose, ToTensor, Normalize
+from tensorboardX import SummaryWriter
```

Comment:
can we wrap this with a try except and raise an error telling the user they need to install the dependency